src/registry/select/select-validation.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import userEvent from '@testing-library/user-event';
+
+import ComponentEditForm from '@/components/ComponentEditForm';
+import {contextRender, screen} from '@/test-utils';
+
+describe('Manual values: must have at least one non-empty value', () => {
+  const component = {
+    id: 'wqimsadk',
+    type: 'select',
+    key: 'select',
+    label: 'A select field',
+    openForms: {
+      dataSrc: 'manual',
+      translations: {},
+    },
+    values: [{value: '', label: ''}],
+    defaultValue: '',
+  };
+  const builderInfo = {
+    title: 'Select',
+    icon: 'th-list',
+    group: 'basic',
+    weight: 70,
+    schema: {},
+  };
+
+  test('Option values and labels are required fields', async () => {
```

Comment:
generally we write the test descriptions so that `describe + test/it` creates a logical English sentence - any way you can improve on that?